.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,52 @@
+name: Scorecard supply-chain security
```

Comment:
are there any existing security concerns that needs to be fixed.

Reply:
I ran a scorecard run using docker run -e GITHUB_AUTH_TOKEN=token gcr.io/openssf/scorecard:stable --show-details --repo=https://github.com/GoogleCloudPlatform/gcsfuse
Overall score is 6.1/10. Please find the results here https://drive.google.com/file/d/1mHM4NWukKBjC504dK17nIM1xnH1Cvo8a/view?usp=sharing&resourcekey=0-fceWBJ2PKYwMSZI9Acjelg